### PR TITLE
Te add local communities

### LINKED ISSUE #14: Other meetups
✨ Cool Internet Webring ✨

`/friends` or `/other-meetups` for example.

Link to:

 - [ ] Cornwall Digital meetup group
 - [ ] Kernow DAT
 - [ ] Software Cornwall
 - [ ] _The_ Digital Plymouth
 - [ ] Plymouth JS
 - [ ] Who else?

### PR DESCRIPTION
**Pull request by tonyedwardspz**
Add local supporting communities to the website

- Adds page
- Adds logos
- Updates CSS

Closes #14 & #15

### CODE REVIEW

**Comment by danreeves** [comment]
Looks great. Think you could add one extra thing to the `.friend li, .organiser li` css block?

```css
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
```

Just to prevent the long urls overflowing onto a new line 👍

**Comment by tonyedwardspz** [comment]
Yarp @danreeves

File: css/style.css [hunk 1]
```diff
@@ -516,45 +516,55 @@ footer nav {
...
+.friend ul {
   list-style: none;
   margin: 0;
   padding: 0;
   margin-bottom: 10px;
   overflow: hidden;
 }
-.organiser li {
+.organiser li,
+.friend li {
```
Comment (danreeves) [inline css/style.css:551]: Add
```css
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
```

**Review by danreeves — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (tonyedwardspz) [reply css/style.css:551]: Done

**Review by danreeves — APPROVED** [review]

**Pull request merged**